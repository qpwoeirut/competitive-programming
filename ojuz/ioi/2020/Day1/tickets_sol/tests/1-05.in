df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
300 1 1
239805936
797626881
420638810
237904102
232188977
310846368
821781473
190745833
199311990
259908864
139781642
167863942
48728614
982027194
4922053
284012728
365696967
544135204
253488363
392928481
41807005
761219264
781518893
221867670
83959163
134809751
123822991
321598858
265968533
281300310
647525916
36520257
231551631
809194857
95237521
944751236
149023519
127946521
320260542
366930233
402525088
121066800
95854764
640011087
272548415
79549851
299470539
441419317
169130120
149661189
579237197
433033247
34299035
856988163
918738173
546877994
491959907
879621753
889000024
732298472
965014284
891224627
574203190
191522092
464698636
260244159
519931301
260852597
177995323
771733732
917243858
859532277
337307160
102071491
856695584
975696703
766663214
106637194
544034907
529854345
317821078
421507279
983983367
707271331
827327123
443363214
602430418
788558963
416445275
865798337
263755303
406676522
411235063
499137433
858662116
552184330
358044481
187314519
674697251
359727051
987427593
441056971
744012340
660430172
402091366
23075757
548977699
216172253
676857342
476875414
426776078
853473609
822588104
679565752
608164131
201523974
596889938
362224218
218444646
599877956
709223184
573903934
991679858
951791089
72394845
9388947
724677760
29444584
592585346
8740935
845301411
486092224
604658916
549015182
575686865
838077491
759706221
253787212
521163328
2287416
184918793
887173509
889276308
998224406
444298478
897945147
569414848
622545875
612534627
555975446
348137619
813285462
614689188
215286668
721949030
481494585
483612634
982435276
117509371
983356725
121672267
214473772
602545950
707655826
609098317
628064531
780051359
40732422
456265755
876367993
256907321
275484824
715747952
820797040
714350746
555305733
286217572
197368379
199575368
803197494
373305766
551191213
107792576
5327812
232763141
176593613
9592159
79356335
601656323
800484175
226707130
487216715
56303800
649856248
906034378
963066825
321731821
978006345
622440252
436702445
948317276
563586423
239004362
838206774
131463293
719790922
943684814
497491697
795186045
791165282
307095987
29339615
759578426
503354036
478574585
804210972
937384727
830045425
27085445
918922122
779357682
570500067
593129730
354511716
342776285
301541791
792017273
392473879
542607991
264675609
191495265
660717027
393793058
483169956
300711250
158892334
996203011
429193506
245772780
896436414
925851098
995361600
646186381
859531631
115904020
752021575
269428914
920977373
571560607
479058484
950624387
369609068
456608226
355201974
677567074
615719071
915820162
440940379
450261069
641647531
783881088
61208676
130003271
386266400
102239015
68158071
909973179
983303924
368805126
420065517
615031367
47723318
860271744
819702553
413379773
347508710
467026584
301386402
389387588
376908401
881679586
615175318
153185949
413146313
914312600
58222354
604043903
580955166
736220366
753264008
671385379
450562239
157302491
472444543
112638782
464935187
938031068
450432373
823831631
397560614
