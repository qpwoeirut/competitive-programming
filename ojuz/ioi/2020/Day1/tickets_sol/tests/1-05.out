131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
73708899206
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
